dali/pipeline/data/types.h

```diff
@@ -214,6 +214,68 @@ inline const char *GetBuiltinTypeName(DALIDataType t) {
   }
 }
 
+inline DALIDataType GetBuiltinDataType(const std::string &type_name) {
```

Comment:
As mentioned, I find the conversion in this direction weird, as this string representation is intended mostly for visual presentation (docs, erros) rather than serialization.

I would suggest sticking with DALIDataType directly.

Reply:
Done